### PR TITLE
File names under the Task folder is changed

### PR DESCRIPTION
**Pull request by thecuriousjuel**
Hello, I have changed all the file names under the task folder from Tasks/*_task.py to Tasks/*.py and have also changed the imports.

### CODE REVIEW

File: state_machine/applications/example/Tasks/battery.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# check for low battery condition
+
+from lib.template import Task
```
Comment (tomyyyD) [inline state_machine/applications/example/Tasks/battery.py:3]: It looks like you updated this import from `lib.template_task` to `lib.template` without changing the name of the file. The filename to change would be found at state_machine/frame/lib/template_task.py

**Review by tomyyyD — CHANGES_REQUESTED** [review]
Thanks for contributing! There was one filename that you forgot to change, see my comment below. Furthermore, we just updated how our file system is structured. We removed the advanced and basic directories and what was the state_machine directory became the root directory. This change likely caused the merge conflicts of your pull request. If you make sure your fork has our most recent commits, it should be pretty easy for you to fix those merge conflicts

**Comment by tomyyyD** [comment]
If you are having issues with the merge conflicts let me know and I can help you out.

File: state_machine/applications/example/Tasks/battery.py (hunk 1 above)
Comment (thecuriousjuel) [reply state_machine/applications/example/Tasks/battery.py:3]: Hello, got the feedback. Working on resolving it.

**Comment by thecuriousjuel** [comment]
Hello, created the new PR for the changes - https://github.com/PyCubed-Mini/flight_software/pull/388

**Comment by thetazero** [comment]
Closed because #388  does this

**Pull request closed without merging**